src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/MobilePay/MobilePayPaymentsResource.cs

```diff
@@ -1,27 +1,28 @@
-﻿using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.MobilePayPayments
-{
-    public class MobilePayPaymentsResource : ResourceBase, IMobilePayPaymentsResource
-    {
-        public MobilePayPaymentsResource(HttpClient httpClient) : base(httpClient)
-        {
-        }
-
-        public Task<MobilePayPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            if (id == null)
-                throw new ArgumentNullException(nameof(id));
-
-            return MobilePayPayment.Get(id, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-
-        public Task<MobilePayPayment> Create(MobilePayPaymentRequest paymentRequest,
-                                                            PaymentExpand paymentExpand = PaymentExpand.None)
-        {
-            return MobilePayPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
-        }
-    }
-}
+﻿using SwedbankPay.Sdk.Payments.MobilePayPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class MobilePayPaymentsResource : ResourceBase, IMobileResource
+    {
+        public MobilePayPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<IMobilePayPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            if (id == null)
+                throw new ArgumentNullException(nameof(id));
+
+            return MobilePayPayment.Get(id, this.HttpClient, GetExpandQueryString(paymentExpand));
+        }
+
+        public Task<IMobilePayPayment> Create(MobilePayPaymentRequest paymentRequest,
+                                                            PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)